docs/guide/why.md

```diff
@@ -14,17 +14,17 @@ We recommend you to read the blog post [Reimagine Atomic CSS](https://antfu.me/p
 
 UnoCSS was started by one of the [Windi CSS](https://windicss.org/)'s team members, with a lot of inspiration taken from the work we did in Windi CSS. While Windi CSS is no longer actively maintained (as of March 2023), you may consider UnoCSS as a *"spiritual successor"* of Windi CSS.
 
-UnoCSS inherits Windi's on-demand natural, [attributify mode](/presets/attributify), [shortcuts](/config/shortcuts), [variant groups](/transformers/variant-group), [compilation mode](/transformers/compile-class) and a lot more. On top of that, UnoCSS is built ground up with the maximum extensibility and performance in mind, making us able to introduce new features like [pure CSS icons](/presets/icons), [valueless attributify](/presets/attributify#valueless-attributify), [tagify](/presets/tagify), [web fonts](/presets/web-fonts), etc.
+UnoCSS inherits Windi's on-demand, natural [attributify mode](/presets/attributify), [shortcuts](/config/shortcuts), [variant groups](/transformers/variant-group), [compilation mode](/transformers/compile-class) and a lot more. On top of that, UnoCSS is built ground up with the maximum extensibility and performance in mind, making us able to introduce new features like [pure CSS icons](/presets/icons), [valueless attributify](/presets/attributify#valueless-attributify), [tagify](/presets/tagify), [web fonts](/presets/web-fonts), etc.
```

Comment:
This was intentional. `on-demand` serves as an adj here. Any better way of expressing that?

Reply:
But as an adjective of what exactly? "Natural" is an adjective itself, hence to me it looked like those two adjectives apply to Windi's attribute mode.

Reply:
Oh, good point, then it should be `on-demand nature` 👍

Reply:
Ah, got it ☺️ Will change it.